src/web/app/pages-instructor/student-list/student-list.component.ts

```diff
@@ -25,6 +20,8 @@ interface StudentDetails {
 })
 export class StudentListComponent implements OnInit {
 
+  readonly DEFAULT_PICTURE_LINK: string = '/assets/images/profile_picture_default.png';
```

Comment:
Is `DEFAULT_PICTURE_LINK` used somewhere in the HTML template? I actually don't know which style do you want to follow:
1. export `DEFAULT_PICTURE_LINK ` to HTML template -> you don't need `setDefaultPic()` -> just use `(error)="student.photoUrl = DEFAULT_PICTURE_LINK;"`

2. Stick to `setDefaultPic()` -> delete export of `DEFAULT_PICTURE_LINK`

Reply:
As of now, I dont have a fixed style. I have changed my code to follow (2) now. I can change it to (1) also if that is the convention. I personally prefer (2) as the method can be extended if needed (perhaps you want to do more actions on error for picture)